ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/capi/cucushift-installer-rehearse-aws-ipi-capi-workflow.metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/capi/cucushift-installer-rehearse-aws-ipi-capi-workflow.yaml",
+	"owners": {
+                "approvers": [
+                        "jhou1",
+                        "sunzhaohua2",
+                        "miyadav",
+                        "huali9"
+                ],
+                "reviewers": [
+                        "jhou1",
+                        "sunzhaohua2",
+                        "miyadav",
+                        "huali9"
+                ]
+        }
```

Comment:
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/29211/pull-ci-openshift-release-master-ci-operator-registry/1534084053718798336 
here shows "msg":"Failed to get registry agent: failed to load registry: failed to load ci-operator registry (failed to load metadata file ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/capi/cucushift-installer-rehearse-aws-ipi-capi-workflow.metadata.json: unexpected end of JSON input)"
seems you miss the last } here

Reply:
it isn't missing in the file , was showing up like this here .. 
`{
        "path": "cucushift/installer/rehearse/aws/ipi/capi/cucushift-installer-rehearse-aws-ipi-capi-workflow.yaml",
        "owners": {
                "approvers": [
                        "jhou1",
                        "sunzhaohua2",
                        "miyadav",
                        "huali9"
                ],
                "reviewers": [
                        "jhou1",
                        "sunzhaohua2",
                        "miyadav",
                        "huali9"
                ]
        }
}
`

Reply:
Added and made commit seems it was missed . Thanks @huali9 , let's see.